frontend/src/store/actions.js

```diff
@@ -48,7 +56,7 @@ export const actions = {
   [LOGOUT]({ commit }) {
     commit(mutations.SET_USER, null);
     deleteToken();
-    window.location.replace(login.path);
+    window.location.replace('/logout_success');
```

Comment:
does it have to be hardcoded?

Reply:
The path could be imported from `routes`, but when I did that  Vuex store could not load for some reason 🤔I think it’s because `VueRouter` init is in the same file as routes - we could store them in different files.

Reply:
That didn't work either :/ I could do this it would be over-engineered. I will leave this as it is.

Reply:
I think that constants need to be refactored to separate file/files. This should  remove circular dependencies and fix this issue.
Let's do it in a separate pr ;)